b2sdk/raw_simulator.py

```diff
@@ -1159,7 +1159,7 @@ class RawSimulator(AbstractRawApi):
     DOWNLOAD_URL_MATCHER = re.compile(
         DOWNLOAD_URL + '(?:' + '|'.join(
             (
-                '/b2api/v[0-9]+/b2_download_file_by_id\?fileId=(?P<file_id>[^/]+)',
+                '/b2api/v[0-9]+/b2_download_file_by_id?fileId=(?P<file_id>[^/]+)',
```

Comment:
wow, this seems like this regexp was invalid from the beginning, right?

instead "fix" should be just adding `r` i.e. `r'...'` as otherwise this `?` will mean `d` is optional

Reply:
good point.